bliss/encoder/variational_dist.py

```diff
@@ -55,20 +57,32 @@ def make_dist(self, x_cat):
         # one with additional distribution parameter groups
         factors = self._parse_factors(x_cat)
         return VariationalDist(
-            factors, self.survey_bands, self.tile_slen, self.include_galaxy_property_loss
+            factors,
+            self.survey_bands,
+            self.tile_slen,
         )
 
 
 class VariationalDist(torch.nn.Module):
     GALSIM_NAMES = ["disk_frac", "beta_radians", "disk_q", "a_d", "bulge_q", "a_b"]
 
-    def __init__(self, factors, survey_bands, tile_slen, include_galaxy_property_loss: bool = True):
+    def __init__(self, factors, survey_bands, tile_slen):
         super().__init__()
 
         self.factors = factors
         self.survey_bands = survey_bands
         self.tile_slen = tile_slen
-        self.include_galaxy_property_loss = include_galaxy_property_loss
+        self.factors_name_set = set(self.factors.keys())
+
+        self.location_factors_name_list = ["loc"]
```

Comment:
How about computing binary class attributes here instead, e.g.,
```
self.loc_available = self._factors_are_available(["loc"])
```
The code is a bit more concise this way and you don't call `self_factors_are_available()` nearly as often.